lib/containers/agda/EverythingContainers.agda

```diff
@@ -0,0 +1,4 @@
+module EverythingContainers where
```

Comment:
I would prefer giving Everything files the same name as the library, i.e. name this `containers.agda`.